src/app/log-in/log-in.component.html

```diff
@@ -1,17 +1,18 @@
 <app-banner></app-banner>
 
-<div class="container home-container" *ngIf="globalVars.inTab || globalVars.webview">
-  <div class="font-weight-bold fs-30px mb-20px">
-    Log in to {{ globalVars.hostname }}
-    <div class="font-weight-normal fs-15px">
+<div class="container home-container text-center" *ngIf="globalVars.inTab || globalVars.webview">
+  <div class="mb-20px">
+    <span class="title-text">Log in to {{ globalVars.hostname }}</span>
+    <div class="main-text">
       Sign up with a new Google account or seed phrase and get {{ globalVars.getFreeDESOMessage() }}!
     </div>
   </div>
 
-  <div class="row">
-    <div class="d-flex flex-column col-sm-6" *ngIf="hasUsers">
-      <ul class="list-group mt-7px mb-30px">
-        <li *ngFor="let item of allUsers | keyvalue" class="list-group-item list-group-item-action cursor-pointer" (click)="selectAccount(item.key)">
+  <div class="d-flex flex-column" *ngIf="hasUsers">
+    <ul class="list-group mt-7px mb-30px saved-seeds-list">
+      <span class="saved-seeds-header d-flex align-items-center"><span>Saved Seeds</span></span>
```

Comment:
Hmmm not sure if I love "Saved Seeds" but I'm struggling to come up with anything better. "Select an account" perhaps?